.travis.yml

```diff
@@ -54,5 +54,7 @@ before_script:
     fi;
 script:
   - java -version
-  - bin/scalafmt --version 0.4.7 --test
+  - bin/install-scalafmt.sh
+  - bin/scalafmt --test
```

Comment:
Is there a way to auto-install, so that you have just a single command that will check if it's installed or not and run the next script? Extra step, means extra step people can and will get wrong.